bin/main

```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts "Hello world!"
+require '../lib/board.rb'
+require '../lib/player.rb'
+require '../lib/winning.rb'
+require '../lib/playagain.rb'
```

Comment:
Use `require_relative` for files within the folder structure. Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.